extensions/transform/org.eclipse.smarthome.transform.jsonpath/README.md

```diff
@@ -1,18 +1,63 @@
 # JsonPath Transformation Service
 
-Extract an element of a JSON string using a [JsonPath expression](https://github.com/jayway/JsonPath#jayway-jsonpath).
+Transforms a JSON structure on basis of the [JsonPath](https://github.com/jayway/JsonPath#jayway-jsonpath) expression to an JSON containing the requested data.
 
-Return `null` if the JsonPath expression could not be found.
+## Examples
 
-## Example
+### Basic Example
 
-Given the JsonPath expression `$.device.status.temperature`:
+Given the JSON
 
-| input | output |
-|-------|--------|
-| `{ "device": { "status": { "temperature": 23.2 }}}` | `23.2` |
+```
+[{ "device": { "location": "Outside", "status": { "temperature": 23.2 }}}]
+```
 
-## Testing Tools
+the JsonPath expression `$.device.location` exstracts the JSON
 
-* [http://jsonpath.com/](http://jsonpath.com/)
-* [http://www.jsonquerytool.com/](http://www.jsonquerytool.com/)
+```
+[ "Outside" ]
```

Comment:
Given the "Differences" description below this would result in `"Outside"`.

Reply:
This is a generall explanation of json and not relate to openhab/smarthome and so i decided to keep it as it would be in json on purpose. 

Same for the next.

Reply:
To me it was confusing to first read this explanation and in the next paragraph see how the result (especially the `[ 23.2 ]`) could posted as an update to a Number item.
I get your point but imho the examples given here should directly reflect the behavior in the framework.